crates/holochain/src/core/workflow/sys_validation_workflow/validate_op_tests.rs

```diff
@@ -197,9 +217,9 @@ async fn validate_valid_create_op() {
     let op = DhtOp::RegisterAgentActivity(fixt!(Signature), Action::Create(create_action));
 
     let outcome = test_case
-        .expect_retrieve_and_retrieve_actions_from_cascade(vec![previous_action])
+        .expect_retrieve_records_from_cascade(vec![previous_action])
```

Comment:
This is where we start to see some results, only hitting cascade once now, not multiple times for the same data